### ISSUE TITLE
Talk to the press

### ISSUE THREAD

**Issue by code-later** [colognerb/refugees#11]
Hi,

my wife is a journalist and after I told her about this project she was eager to offer her help in covering the whole thing. This would eventually result in radio pieces and online articles. She is working mainly for WDR, DLF and Deutsche Welle. If you're interested in this contact her via info@raynabreuer.com.

**Comment by erdnusskeks** [comment]
Sounds great! Thanks, Dirk!

**Comment by koos** [comment]
Talk to @Schwizzle - he can write down a press release.

**Comment by Schwizzle** [comment]
Are you still interested? Don't hesitate to contact me!

**Comment by jhilden** [comment]
yes, we are super interested.  the first basic website is now online and I would think that it should contain all the necessary information for a press release:  http://cologne.refugeesonrails.org/

also, if you want to change any text on the site, feel free to do it.  you can find it here:  https://github.com/colognerb/refugees/tree/gh-pages

**Comment by jhilden** [comment]
@Schwizzle is there anything else you need?  It would be good to send out something in the near future

**Comment by erdnusskeks** [comment]
I had a look at the press release. I think it's fine and should be released as soon as possible. : ) 
@Schwizzle Do you have a list of all receivers? So that we know who is already informed. Maybe we can even add some more receivers to your list like @railsbros-dirk's wife Rayna.

**Comment by Schwizzle** [comment]
I Sent it to an existing list of multiplicators I recently started to
build up. I can show you on tuesday which receivers are included in that.

Best, Michael

Am Freitag, 30. Oktober 2015 schrieb Denise :

> I had a look at the press release. I think it's fine and should be
> released as soon as possible. : )
> @Schwizzle https://github.com/Schwizzle Do you have a list of all
> receivers? So that we know who is already informed. Maybe we can even add
> some more receivers to your list like @railsbros-dirk
> https://github.com/railsbros-dirk's wife Rayna.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/colognerb/refugees/issues/11#issuecomment-152528610.

## 

Von Gmail Mobile gesendet

**Comment by jhilden** [comment]
@Schwizzle awesome.  Thanks a lot for that.

btw:  What is happening on tuesday?  (our orga meetings are on Thursday)

**Comment by oezlemburan** [comment]
Hi Everyone, 

My name is Oezlem and I'm part of the //Refugees on Rails Team in Berlin. I happen to be around (well, in Osnabrück) and look forward to meeting you all next week in Cologne! As discussed with Jakob, an editor from Innovation Stories, Joe Dodgshun, (http://innovationsstories.com/) will also come along. If there is anything I can help you with until then (remotely), let me know! You reach me best via oezlem@refugeesonrails.org

Thank you for supporting //RoR and starting the first local chapter, by the way! You guys rock!
